### DISCUSSION TITLE
Hide redundant versions in pull-request badge table

### DISCUSSION THREAD

**Discussion by Scrat94** [renovatebot/renovate#28110] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Hi, is it possible to hide redundant versions? I have two scenarios where I would like to have a "cleaner" badge table view in the pull request / merge request:

Scenario 1) I have python package X with version 1.2.3. I also have a customManager that updates the version 1.2.3 in a README.md via regex. Now, both versions will show up in the table view despite having the same source

Scenario 2) I have a python package Y and use it also in pre-commit hooks. The version of the python package and the pre-commit package are always in sync. Also both show up in the table view generating some noise. 

See screenshot:
![2024-03-25_003725_msedge](https://github.com/renovatebot/renovate/assets/31823370/37db7ad7-d267-41e6-92b0-6df3b2ea9577)

The screenshot has 6 dependencies which are redundant (3x2same versions) So instead of 7 dependencies in this screenshot, ideally I would like to show only 4. 

If this feature does not exist, I would love to see, if there would be a flag for customManager / pre-commit to say "hide from badge table view: yes/no". 

Thanks a lot in advance!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
No, because the updates are not identical. There's different currentValue and/or depType
